### PR TITLE
sql to create users table and update of spaces table

### PR DESCRIPTION
**Pull request by nepeanwjdw**
Co-Authored-By: Paul Fazackerley <alittlecross@live.co.uk>

### CODE REVIEW

File: db/migrations/02_create_spaces_table.sql [hunk 1]
```diff
@@ -1 +1,7 @@
-CREATE TABLE spaces(id SERIAL PRIMARY KEY, username VARCHAR(50), description VARCHAR(300), price INT);
+CREATE TABLE spaces(
+  id SERIAL PRIMARY KEY,
```
Comment (StuAA78) [inline db/migrations/02_create_spaces_table.sql]: Thinking about it, maybe we would be better off having the id in the spaces table be space_id, and the id in the users table user_id just so we don't have 2 "id" columns that mean different things?

File: db/migrations/03_create_users_table.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50),
+  email char(254) UNIQUE,
```
Comment (stephanierosep) [inline db/migrations/03_create_users_table.sql]: I think email might need changing to VARCHAR, as char is a fixed length data type

Comment (nepeanwjdw) [reply db/migrations/03_create_users_table.sql]: thanks Steph, I have made the changes as requested

File: db/migrations/02_create_spaces_table.sql (hunk 1 above)
Comment (nepeanwjdw) [reply db/migrations/02_create_spaces_table.sql]: thanks Stu, I have made a new commit which takes into account your comments

**Pull request merged**